zds/settings/abstract_base/zds.py

```diff
@@ -68,6 +68,9 @@
         "la communauté.",
         "googleAnalyticsID": zds_config.get("google_analytics_id"),
         "googleTagManagerID": zds_config.get("google_tag_manager_id"),
+        "matomoSiteID": zds_config.get("matomo_site_id", 3),
+        "matomoUrl": zds_config.get("matomo_url", "https://matomo.zestedesavoir.com"),
+        "matomoTokenAuth": zds_config.get("matomo_token_auth", "b8f815cd5ad1e90af9d03c1dd8a544d5"),
```

Comment:
Pour moi on ne devrait pas mettre un token d'auth dans git, là ça rend publique notre machine et donc nous expose à du ddos.

Reply:
On peut aussi supprimer les paramètres `googleAnalyticsID` et `googleTagManagerID` qui ne sont plus utiles, je suppose ?

Reply:
Pour les paramètres Google, je réserve leur suppression dans une deuxième itération quand on aura bien observé matomo en prod. La deuxième itération viendra avec la suppression de Google partout, parce que pour le moment, on demande encore le consentement par exemple. Je préfère séparer les deux PR pour éviter le gros big bang.